### PR TITLE
initial documentation

### PR DESCRIPTION
**Pull request by pshemass**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,12 @@
 | --- | --- | --- | --- |
 | [![Build Status][Badge-Circle]][Link-Circle] | [![Release Artifacts][Badge-SonatypeReleases]][Link-SonatypeReleases] | [![Snapshot Artifacts][Badge-SonatypeSnapshots]][Link-SonatypeSnapshots] | [![Badge-Discord]][Link-Discord] |
 
-ZIO-based cluster membership and failure detection.
+ZIO-based liabrary for cluster membership and failure detection using gossip based protocol.
```
Comment (softinio) [inline README.md]:
```suggestion
ZIO-based library for cluster membership and failure detection using gossip based protocol.
```

File: README.md [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
+
+This library is inspired by Hashicorp research on improving [SWIM](https://ieeexplore.ieee.org/document/1028914) protocol published in [Lifeguard](https://ieeexplore.ieee.org/document/1028914) paper. 
+
+Please find more details on our [website](https://zio.github.io/zio-memberlist/).
 
-> This library is still a work-in-progress. Watch this space!
 
 # License
 [License](LICENSE)
```
Comment (softinio) [inline README.md:15]: Please add apache2 license as per other projects

**Review by softinio — CHANGES_REQUESTED** [review]
- Left you a few suggestions for you to review.
- Can you amend: https://github.com/zio/zio-memberlist/blob/master/docs/about/contributing.md replacing all mentions of `zio` with `zio-memberlist` also? 

Thanks. 

ps: Great project excited for this 👏

**Comment by softinio** [comment]
@adamgfraser added you as review as a heads up, in case you need to add CLA etc for this repo. Thanks.

File: README.md (hunk 2 above)
Comment (pshemass) [reply README.md:15]: what do you mean? project was created from template and Apache 2.0 is in it https://github.com/zio/zio-memberlist/blob/master/LICENSE

Comment (softinio) [reply README.md:15]: AH yes my mistake! Sorry about that.

File: README.md (hunk 1 above)
Comment (pshemass) [reply README.md]: `contribution.md` adjusted, README.md typo fixed.

ps. I'm excited too :)

**Review by softinio — APPROVED** [review]
LGTM 👏

**Pull request merged**